Ada_Zangemann-cs.txt

```diff
@@ -186,784 +186,712 @@ Society should support those who respect others' freedom.
+Nahoře je kulatý bílý znak s kopečkem cukrově růžové zmrzliny v kornoutku.
+Na zadní straně stroje je chladicí zařízení s písmenem „Z“ jako logem Zangemanna.
 ]]
-Adults and children loved his inventions. All the kids wanted to have 
-one of his skateboards — complete with the latest sounds to show off to 
-their friends at school. Kids also loved his speaker invention, which 
-could instantly play any song you requested. And, of course, everyone 
-loved the ice cream served every afternoon from the automatic, 
-extra-delicious Original Zangemann Ice Cream Machine. It all seemed like 
-magic, but the secret was simple: the little computers that Zangemann 
-built into his machines made it all possible.
+Dospělí i děti jeho vynálezy milovali. Děti chtěly mít Zangemannův skateboard a a k němu pokaždé ten nejnovější zvuk, aby se s ním mohly chlubit kamarádům na školním dvoře. Mnozí měli také jeho praktické reproduktory. Stačilo do nich jen uvést název písně a ta se hned začala přehrávat. Komu ještě nějaké kapesné zbylo, mohl si odpoledne koupit automaticky namíchanou zmrzlinu z originální Zangemannovy zmrzlinovačky. Vypadalo to jako kouzla, ale vše zařizovaly malé počítače, které Zangemann do svých přístrojů vestavěl.
 [[I:ada-p08a
-IMAGE: An original Zangemann electric skateboard leaving a blue trail 
-with musical notes and colored circles. Only the knees, white socks, and 
-olive green shoes with white soles of the child riding it are shown. 
+OBRÁZEK: Originální Zangemannův elektrický skateboard zanechávající modrou stopu s notami a barevnými kruhy. Z dítěte, které jede na skateboardu, jsou vidět jenom koleny, bílé ponožky a olivově zelené boty s bílou podrážkou.
 ]]
 
 ** 9 **
 
 [[I:ada-p09
-IMAGE: Ada tinkers with objects in her improvised lab.
-Ada is sitting on an overturned water case, and working on a repair with 
-a yellow screwdriver. Also on the table are green pliers, a red hammer, 
-a white power strip, and another gadget. Ada is at a table consisting of 
-a wooden board supported by two oil drums (one gray and one red) and a 
-small wooden sawhorse. Behind Ada there are a washing machine, pipes, 
-and a board. In front of the improvised table there are tires and an old 
-phone with faulty wire. Alan is coming toward Ada holding a yellow fan 
-over his head. Alan now wears white socks and yellow shoes. 
+OBRÁZEK: Ada kutí s předměty ve své improvizované laboratoři.
+Ada sedí na převrácené bedně s vodou a pracuje na opravě s pomocí žlutého šroubováku. Na stole jsou také zelené kleště, červené kladivo, bílá prodlužovací šňůra a další přístroj. Ada pracuje u stolu, který se skládá z dřevěné desky položené na dvou olejových sudech (jednom šedém a jednom červeném) a malém dřevěném kozlíku. Za Adou je pračka, potrubí a deska. Před improvizovaným stolem jsou pneumatiky a starý telefon s vadným drátem. Alan přichází k Adě a nad hlavou drží žlutý větrák. Alan má nyní na sobě bílé ponožky a žluté boty.
```

Comment:
Nevidím ten obrázek, tak nevím, ale fakt jsem se zarazila u "sedí na převrácené bedně s vodou" a přemýšlela, jak to, že se ta voda nevylije, když je bedna převrácená. Při přečtení anglické verze si představím spíš převrácený kanystr na vodu (ne nutně s vodou). Nevím, jestli jsou někde obrázky zveřejněny, ale podle popisu jsem našla https://pedagogie.ac-clermont.fr/numerique-educatif/wp-content/uploads/sites/6/2024/10/ADA-2.jpg a podle něj bych to popsala spíš jako ten kanystr na vodu.

Reply:
Ok, to dává smysl.

Reply:
Souhlasím, přepravka je lepší. Asi i na tom obrázku je přepravka, jen mi to nedocvaklo.

Reply:
Obrazky jsou zde a konkretne tento https://github.com/xsuchy/ada-zangemann-cz/blob/main/illustrations/common/ada-p04.png